packages/fast-layouts-react/src/column/column.tsx

```diff
@@ -91,7 +92,7 @@ export class Column extends Foundation<ColumnHandledProps, ColumnUnhandledProps,
      */
     private getValueByBreakpoint<T>(breakpointSet: T[]): T {
         if (!canUseDOM()) {
-            return breakpointSet[0];
+            return breakpointSet[this.props.defaultBreakpoint];
```

Comment:
This doesn't guard that the default breakpoint won't be outside the array. i.e. breakpointSet could be of size 5, and this.props.defaultBreakpoint could be 20.
It would be better to remove the canUseDOM check there and instead do

    const breakpoint: Breakpoint = canUseDOM()
            ? identifyBreakpoint(window.innerWidth)
            : this.props.defaultBreakpoint;

and let the slice-pop do its thing